src/main/java/homework/seven/ArrayValueCalculator.java

```diff
@@ -0,0 +1,30 @@
+package homework.seven;
+
+public class ArrayValueCalculator {
+    String[][] array = new String[4][4];
+
+    public int doCalc(String[][] array) throws ArraySizeException, ArrayDataException {
+        int suma = 0;
+        this.array = array;
+        if (this.array.length > 4) {
+            throw new ArraySizeException();
+        } else {
+            for (int i = 0; i < 4; i++) {
```

Comment:
What happen if we have 3 rows?

Reply:
I fixed it array.length != 4 and now if the size is small or more it will give an error